server/Dockerfile

```diff
@@ -18,8 +18,12 @@ COPY --from=builder /opt /opt
 # using already installed packages from builder for faster build time
 COPY --from=builder /kadalu /kadalu
```

Comment:
Should we change the base image also to same? that way, we don't have issues with glusterfs build, as its built from source, and better to keep the base and the target prod images in same version.

Reply:
We can give a try, in that case [this](https://hub.docker.com/layers/debian/library/debian/buster/images/sha256-41d7d0aeed1b4ff2e9041096601e8c1e5cdbffc7bf2c265f03b22ad39a3fa2ff?context=explore) is a better choice because component images are based on debian buster and having builder based on same will be good.

Let me push a commit and see what CI has to say.